src/main/resources/mappings/workflow-steps.json

```diff
@@ -92,5 +92,29 @@
             "model_id",
             "register_model_status"
         ]
+    },
+    "register_agent": {
+        "inputs":[
+            "name",
+            "type",
+            "llm",
+            "tools",
+            "parameters",
+            "memory",
+            "created_time",
+            "last_updated_time",
+            "app_type"
+        ],
+        "outputs":[
+            "agent_id"
+        ]
+    },
+    "create_tool": {
```

Comment:
should we have more inputs here

Reply:
We just add mandatory inputs here for validation. Rest are optional from [here](https://github.com/opensearch-project/ml-commons/blob/feature/agent_framework_dev/common/src/main/java/org/opensearch/ml/common/agent/MLToolSpec.java#L44)